<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextClassTest</class>
 <widget class="QWidget" name="TextClassTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBoxText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>First default string</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Second default string</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Third default string</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить выбранную строку</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAddString">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAddIndex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAddString">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Добавить в начало</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Добавить в конец</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Добавить по индексу</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_find">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_found">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_found">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Заменить на&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Количество найденных&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_place">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_find_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_replace">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_replacements">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>380</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Поиск&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Найденные&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Найти&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_find">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Количество замен&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addString">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Длина текста&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_strSize">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
